src/app/api/stripe/sync.ts

```
@@ -24,9 +28,51 @@ export async function sync_stripe_data(
+        limit: 1,
+        status: 'all',
+        expand: ['data.default_payment_method']
+    })
+
+    if (subscriptions.data.length === 0) {
+        const subData = { status: 'none' } satisfies StripeSubData
+        await redis.set(get_redis_key('stripe:customer', customer_id), subData)
+        return subData
+    }
+
+    const subscription = subscriptions.data[0]!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)